### PR TITLE
Added Responsiveness and fixed some minor css changes

### PR DESCRIPTION
**Pull request by Rohanmond**
Fixed reponsivensss
Added add / remove wishlist in cart page
remove all class and convert to className
Added close open button for filters in small screen
Refractored product card
Refractored cart card
Refractored home page images

Preview Link: https://fashi-fy-react-git-responsiveness-rohanmond.vercel.app/

### CODE REVIEW

**Review by hrishi82 — APPROVED** [review]
code looks good to me, can be merged!

**Review by ScoobyDubeyDo — APPROVED** [review]
The code looks good, Liked the good naming conventions

**Review by anuj-kumary — APPROVED** [review]
- Clean code
- Good naming conventions used

File: fashify/src/pages/Auth/Logout/Logout.jsx [hunk 1]
```diff
@@ -3,21 +3,21 @@ import '../Auth.css';
...
           >
             See products
           </button>
         </div>
-        <div class='img-responsive-container obj-pos-tp-ct logout-img-container'>
+        <div className='img-responsive-container obj-pos-tp-ct logout-img-container'>
           <img
-            class='img-responsive'
+            className='img-responsive'
             // src="https://res.cloudinary.com/donqbxlnc/image/upload/v1647812850/logout_re_pfrpun.png"
```
Comment (znjs) [inline fashify/src/pages/Auth/Logout/Logout.jsx:21]: can remove the commented code

File: fashify/src/pages/Cart/components/CartCard/CartCard.jsx [hunk 2]
```diff
@@ -68,10 +78,56 @@ const CartCard = ({ el }) => {
...
+          payload: { wishlist: res.data.wishlist },
+        });
+        if (wished) {
+          ToastHandler(ToastType.Warn, 'Deleted from wishlist');
+        } else {
+          ToastHandler(ToastType.Success, 'Added to wishlist');
+        }
+      }
+    } catch (err) {
+      console.log(err);
```
Comment (znjs) [inline fashify/src/pages/Cart/components/CartCard/CartCard.jsx:105]: can use `console.error` to give a semantic meaning

File: fashify/src/pages/ProductList/ProductList.css [hunk 3]
```diff
@@ -9,7 +9,7 @@
   display: none;
 }
 .dd_menu::before {
-  content: "";
+  content: '';
   position: absolute;
   top: -20px;
   right: 55px;
```
Comment (znjs) [inline fashify/src/pages/ProductList/ProductList.css:15]: can use relative units

**Review by znjs — APPROVED** [review]
- clean code
- Good naming conventions used

**Pull request merged**